src/quo/components/settings/category/settings/view.cljs

```diff
@@ -4,23 +4,21 @@
     [quo.components.settings.category.style :as style]
     [quo.components.settings.settings-item.view :as settings-item]
     [quo.theme :as quo.theme]
-    [react-native.core :as rn]))
+    [react-native.core :as rn]
+    [react-native.pure :as prn]))
```

Comment:
Hey @flexsurfer, thanks for the initiative :) I see WIP in the title so please lmk if I my comment should be ignored as I have some other thoughts. 

one thing to note `prn` is shadowing this method https://cljs.github.io/api/cljs.core/prn. 
imo it would be nicer to have something more verbose to begin with as this is all quite new and our codebase can be confusing enough for external/new contributors. Perhaps `pure-rn` 🤔

Reply:
also is it possible that we can use `[` `]` brackets instead? of `(` `)`? or is there a specific reason we are using those brackets here instead?

Reply:
> Hey @flexsurfer, thanks for the initiative :) I see WIP in the title so please lmk if I my comment should be ignored as I have some other thoughts.
> 
> one thing to note `prn` is shadowing this method https://cljs.github.io/api/cljs.core/prn. imo it would be nicer to have something more verbose to begin with as this is all quite new and our codebase can be confusing enough for external/new contributors. Perhaps `pure-rn` 🤔

yes, thanks. its just a quick implementation for experiments, for sure it should be named differently :)

Reply:
> also is it possible that we can use `[` `]` brackets instead? of `(` `)`? or is there a specific reason we are using those brackets here instead?

Do you mean hiccup? yes good question, we all should think on that , here will be more details https://www.notion.so/Reagent-b1495d7e76204be1b771f93680b5fbf0?pvs=4